Framework/PythonInterface/plugins/algorithms/SNSPowderReduction.py

```diff
@@ -336,7 +336,8 @@ def PyExec(self):  # noqa
         # get the user-option whether an existing event workspace will be reloaded or not
         reload_event_file = self.getProperty('ReloadIfWorkspaceExists').value
 
-        if self.getProperty("Sum").value:
+        if self.getProperty("Sum").value and len(samRuns) > 1:
+            self.log().information('Ignoring value of "Sum" property')
```

Comment:
How is the "Sum" property ignored here?

Reply:
It is only ignored when number of runs to sum is 1.

Reply:
@wdzhou is correct. They were "summing" one file which adds a block against filtering events. It is a user error that is being ignored.